script.js

```diff
@@ -1,12 +1,375 @@
+    if (currentShow.summary === null || currentShow.summary === '') {
+        showSummary.innerText = 'The summary of this show is coming soon.'
+    } else {
+        showSummary.innerText = currentShow.summary.replace(/<\/?[^>]+(>|$)/g, '')
+    }
+    summaryDiv.appendChild(showSummary)
+
+    let detailsDiv = createHtmlElement('div')
+    detailsDiv.className = 'current-show-info'
+    containerShow.appendChild(detailsDiv)
+    let rated = createHtmlElement('p')
+    rated.innerText = `Rating: ${currentShow.rating.average}`
+    detailsDiv.appendChild(rated)
+    let genre = createHtmlElement('p')
+    genre.innerText = `Genres: ${currentShow.genres.join('|')}`
+    detailsDiv.appendChild(genre)
+    let status = createHtmlElement('p')
+    status.innerText = `Status: ${currentShow.status}`
+    detailsDiv.appendChild(status)
+    let runtime = createHtmlElement('p')
+    runtime.innerText = `Runtime: ${currentShow.runtime}`
+    detailsDiv.appendChild(runtime)
+}
+//fetch show episodes
+function loadShowEpisodes() {
+    let selectedUrl = `https://api.tvmaze.com/shows/${selectedShowInfo.id}/episodes`
+    fetch(selectedUrl)
+        .then((response) => response.json())
+        .then((data) => {
+            allEpisodes = data
+            makePageForEpisodes(allEpisodes)
+            selectInputLoad(allEpisodes)
+        })
+        .catch((error) => console.log(error))
 }
 
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+    for (let i = 0; i < episodeList.length; i++) {
+        renderSingleEpisode(episodeList[i])
+    }
+    searchFunc()
 }
 
-window.onload = setup;
+function renderSingleEpisode(episode) {
+    // Creating a div element for each episode
+    let divElement = createHtmlElement('Div')
+    divElement.className = 'episodeContainer'
+    divContainer.appendChild(divElement)
+        // Adding h1 to it as a title
+    let h1Elem = createHtmlElement('h1')
+    h1Elem.innerHTML =
+        episode.name +
+        ' - S' +
+        episode['season'].toString().padStart(2, '0') +
+        'E' +
+        episode['number'].toString().padStart(2, '0')
+    divElement.appendChild(h1Elem)
+        // Adding an image to the episode
+    let imageElem = createHtmlElement('img')
+    imageElem.className = 'episode-image'
+    if (imageElem.image === null || imageElem.image === '') {
+        imageElem.src = 'https://dozenpixels.com/static/img/blog/coming-soon.png'
+    } else {
+        imageElem.src = episode.image.medium
+    }
+
+    divElement.appendChild(imageElem)
+        // Adding a paragraph to it as a summary of the episode
+    let paragraph = createHtmlElement('P')
+    paragraph.className = 'episode-summary'
+    episode.summary === null || episode.summary === '' ?
+        (paragraph.innerText = ' The summary of this episode is coming soon.') :
+        (paragraph.innerHTML = episode.summary)
```

Comment:
you used here innerHTML

Reply:
good spot I thought I have updated it with innerText with replace method